packages/api/src/shared/config.ts

```diff
@@ -260,8 +260,12 @@ export class Config {
     return getEnvVar("FHIR_TO_MEDICAL_RECORD_LAMBDA_NAME");
   }
 
-  static getDocQueryResultsLambdaName(): string {
-    return getEnvVarOrFail("DOC_QUERY_RESULTS_LAMBDA_NAME");
+  static getOutboundDocumentQueryRespsLambdaName(): string {
+    return getEnvVarOrFail("OUTBOUND_DOC_QUERY_RESPS_LAMBDA_NAME");
+  }
+
+  static getOutboundDocRetrievalRespsLambdaName(): string {
+    return getEnvVarOrFail("OUTBOUND_DOC_RETRIEVAL_RESPS_LAMBDA_NAME");
```

Comment:
nit: would `OUTBOUND_DOC_RETRIEVAL_RESP_LAMBDA_NAME` be better? AFAIK we don't use `RESPS` terminology anywhere else